lottie/src/main/java/com/airbnb/lottie/animation/content/FillContent.java

```diff
@@ -120,6 +121,13 @@ public FillContent(final LottieDrawable lottieDrawable, BaseLayer layer, ShapeFi
       dropShadowAnimation.applyTo(paint);
     }
 
+    // Set layer paint blend mode
+    final BlendModeCompat blendMode = layer.getBlendMode().toNativeBlendMode();
+    if (blendMode != null) // PaintCompat.setBlendMode accepts null but no surprises please
```

Comment:
Is there a specific reason you don't want to pass in null here? Looking at the PaintCompat code, I think it would be fine.